src/Device/Driver/AirControlDisplay.cpp

```diff
@@ -68,8 +69,27 @@ ParsePAAVS(NMEAInputLine &line, NMEAInfo &info)
     unsigned volume;
     if (line.ReadChecked(volume))
       info.settings.ProvideVolume(volume, info.clock);
+  } else if (type == "XPDR"sv) {
```

Comment:
bad indent?

Reply:
I checked and it looks good to me. Maybe because the if-sentence above is not wrapped in curly braces?